setup.py

```diff
@@ -46,6 +46,7 @@
         "ismrmrd==1.9.1",
         "tensorboard>=2.5.0",
         "tqdm",
+        "sewar",
```

Comment:
Perhaps don't add sewar, it's only for the VIF. You can use a mechanism such as https://github.com/NKI-AI/dlup/blob/main/dlup/utils/imports.py to check if it is there and otherwise give a `RunTimeError` when you try to run the Calgary Campinas or something like that?

Reply:
No I don't really agree as you may want to run C-C but withouf VIF. Maybe if "calgary_campinas_vif_metric" is requested as an evaluation metric.

Reply:
Yes. But then it must crash before it is at the validation step. E.g. when building metrics.